routes.js

```diff
@@ -0,0 +1,48 @@
+const express = require('express');
+const app = express();
+const port = 5000;
```

Comment:
Consider using environment variables for sensitive or configurable information like the port number. This will make your application more secure and configurable. [important]